src/detectors/BEMC/Cluster_factory_EcalBarrelScFiClusters.h

```diff
@@ -42,7 +42,7 @@ class Cluster_factory_EcalBarrelScFiClusters : public JFactoryT<edm4eic::Cluster
         // for endcaps.
         m_enableEtaBounds=false;//{this, "enableEtaBounds", false};
 
-
+        app->SetDefaultParameter("BEMC:EcalBarrelScFiClusters:input_protoclust_tag", m_input_protoclust_tag, "Name of input collection to use");
```

Comment:
duplicated on line 50...though this has the description

Reply:
Good catch. I did not see that buried there. That will be an issue when it is set to a different value since JANA will see it already has a default and the second call to SetDefaultParameter does not match it. The second line needs to be removed. I will do that.